docs/demo/gaiden-css/components/card/centered.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #eff4fa;
```

Comment:
@thiagodelmotte this is only a docs file. The point here is to evidence the cards shadow effect on the page.